test/Generics/existential_restrictions.swift

```diff
@@ -8,14 +8,14 @@ protocol CP : class { }
   static func createNewOne() -> SP
 }
 
-func fP<T : P>(_ t: T) { }
+func fP<T : P>(_ t: T?) { }
```

Comment:
Is the addition of '?' in this file intentional?

Reply:
Yes, I made these parameters optional so the diagnostics stay the same later in the file (instead of disappearing due to opening)